doc/data/messages/t/trailing-whitespace/bad.py

```diff
@@ -0,0 +1 @@
+print("Hello")  # [trailing-whitespace]  
```

Comment:
```suggestion
print("Hello")  # [trailing-whitespace]   
                #                      ^^^ whitespaces
```

Do we want to mark whitespaces like this?

Reply:
Good idea, that could be helpful when reading the doc. Especially since I don't know if sphinx is not automatically removing trailing whitespaces.

Reply:
```suggestion
print("Hello")  # [trailing-whitespace]   
#                                                   ^^^ trailing whitespaces
```